builtin/v9/verifreg/verified_registry_state.go

```diff
@@ -214,3 +214,77 @@ func (st *State) LoadClaimsToMap(store adt.Store, providerIdAddr address.Address
 
 	return goMap, nil
 }
+
+func (st *State) GetAllClaims(store adt.Store) (map[ClaimId]Claim, error) {
```

Comment:
I had implemented these because I thought I needed them. They basically got copied into `verifreg/invariants.go`, but with some added logic in the middle like `checkAllocationState` `checkClaimState`.